dev-server/documents/html/shadow-dom.mustache

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Hypothesis Client Test</title>
+<style>
+  body {
+    font-family: sans-serif;
+  }
+</style>
+  <template id="my-paragraph">
+    <style>
+      p {
+        color: white;
+        background-color: #666;
+        padding: 5px;
+      }
+     </style>
+     <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est. Mauris placerat eleifend leo.</p>
+  </template>
+  <script>
+    customElements.define('my-paragraph',
+      class extends HTMLElement {
+        constructor() {
+          super();
+          let template = document.getElementById('my-paragraph');
+          let templateContent = template.content;
+    
+          const shadowRoot = this.attachShadow({mode: 'open'})
+            .appendChild(templateContent.cloneNode(true));
+        }
+      }
+    );
+  </script>
+</head>
+<body>
+  <h2>Shadow DOM example</h2>
+  <my-paragraph></my-paragraph>
```

Comment:
Or...maybe even add some content to the page itself (as HTML) to explain what it's designed to allow testing of?

Reply:
I'll do.